vnext/local-cli/runWindows/utils/build.js

```diff
@@ -65,51 +65,25 @@ async function nugetRestore(nugetPath, slnFile, verbose, msbuildVersion) {
 }
 
 async function restoreNuGetPackages(options, slnFile, verbose) {
-  let nugetPath =
-    options.nugetPath || path.join(os.tmpdir(), 'nuget.4.9.2.exe');
+  const nugetPath = path.join(os.tmpdir(), 'nuget.4.9.2.exe');
```

Comment:
>4.9.2 [](start = 50, length = 5)

nit: Since this is in botht he filename and the url, perhaps extract to a const at the top of the file, named identical to others so if we want to upgrade it would be easy to identify all nuget versions used.